### PR TITLE
Load new photos dynamically

### PR DESCRIPTION
**Pull request by ojarva**
Implementation by @phadej.

### CODE REVIEW

File: routes/photo.js [hunk 1]
```diff
@@ -42,7 +42,12 @@ module.exports = function(db) {
 		var collection = db.get('photocollection');
 		collection.find({},{}, function(e, docs){
 			_(docs).reverse();
-			res.send(JSON.stringify(docs));
+			var body = JSON.stringify(docs);
+			res.writeHead(200, {
+				"Content-Length": body.length,
+				"Content-Type": "application/json"
+			});
+			res.send(body);
```
Comment (phadej) [inline routes/photo.js:50]: uuh, we use tabs here

Comment (ojarva) [inline routes/photo.js:50]: @phadej This comment is a bit misplaced (or I'm missing something here), but fixed photos.js to use tabs instead of spaces.

Comment (phadej) [inline routes/photo.js:50]: I meant otherwise. Didn't we use spaces in this project mostly? Sorry for the confusion.

Comment (ojarva) [inline routes/photo.js:50]: Nope. At least some files (routes/photo.js, routes/selfie.js) use tabs. I'll create another pull request to fix that.

**Pull request merged**